provision.yml

```diff
@@ -206,6 +206,7 @@
     - { role: "letsencrypt",          tags: ["demo1","demo-letsencrypt"] }
     - { role: "demo-etherpad",        tags: ["demo1","demo-etherpad"]  }
     - { role: "demo-weblogin",        tags: ["demo1","demo-weblogin"]  }
+    # - { role: "demo-wordpress",       tags: ["demo1","demo-wordpress"]  } # Breaks CI
```

Comment:
Could you please remove this?  If it breaks the CI, we should properly fix it.

Reply:
We can't. CI is too small